gridstatus/ieso.py

```diff
@@ -1045,6 +1045,10 @@ def get_hoep_real_time_hourly(
                 )
             raise
 
+        # On this date, IESO published duplicates for hour ending instead of going 1-24
```

Comment:
Duplicates published on 2025-04-16

<img width="514" alt="Screenshot 2025-04-17 at 07 35 41" src="https://github.com/user-attachments/assets/80d82c86-4c63-405b-94b7-8347433fd235" />

https://reports-public.ieso.ca/public/DispUnconsHOEP/PUB_DispUnconsHOEP_20250416.xml

Reply:
Assuming the rows are in order, we can just use the index as the correct hour endings.

Reply:
I actually don't think this is correct assumption if you look at the generated time of the report.

given the report stops at 19:00 when the day as completed is suspect as well